code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -884,3 +884,47 @@
 
 /datum/reagent/toxin/mimesbane/on_mob_delete(mob/living/L)
 	L.remove_trait(TRAIT_EMOTEMUTE, id)
+
+/datum/reagent/toxin/bonehurtingjuice //oof ouch
+	name = "Bone Hurting Juice"
+	id = "bonehurtingjuice"
+	description = "A strange substance that looks a lot like water. Drinking it is oddly tempting. Oof ouch."
+	color = "#AAAAAA77" //RGBA: 170, 170, 170, 77
+	toxpwr = 0
+	taste_description = "bone hurting"
+	overdose_threshold = 50
+	metabolization_rate = 2
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_add(mob/living/carbon/M)
+	M.say("oof ouch my bones", forced = "bonehurtingjuice")
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_life(mob/living/carbon/M)
+	M.adjustStaminaLoss(8, 0)
+	if(prob(20))
+		switch(rand(1, 3))
+			if(1)
+				var/list/possible_says = list("oof", "ouch", "my bones", "oof ouch", "oof ouch my bones")
+				M.say(pick(possible_says), forced = "bonehurtingjuice")
+			if(2)
+				var/list/possible_mes = list("oofs softly", "looks like their bones hurt", "grimaces, as though their bones hurt")
+				M.say("*custom " + pick(possible_mes), forced = "bonehurtingjuice")
+			if(3)
+				to_chat(M, "<span class='warning'>Your bones hurt!</span>")
+	return ..()
+
+/datum/reagent/toxin/bonehurtingjuice/overdose_process(mob/living/M)
+	if(prob(4) && iscarbon(M)) //big oof
+		var/stamina_damage = 200
+		switch(rand(1, 4)) //God help you if the same limb gets picked twice quickly.
+			if(1)
+				M.get_bodypart(BODY_ZONE_L_ARM).receive_damage(0, 0, stamina_damage)
+			if(2)
+				M.get_bodypart(BODY_ZONE_R_ARM).receive_damage(0, 0, stamina_damage)
+			if(3)
+				M.get_bodypart(BODY_ZONE_L_LEG).receive_damage(0, 0, stamina_damage)
+			if(4)
+				M.get_bodypart(BODY_ZONE_R_LEG).receive_damage(0, 0, stamina_damage)
```

Comment:
Don't use `.` after procs. It's equivalent to using `:`

Reply:
What you should do is assign the bodypart enum to a var in the switch and then do 
```dm
var/obj/item/bodypart/bp = M.get_bodypart(selected_part)
bp.receive_damage(0, 0, stamina_damage)
```

Reply:
I see what you mean now, that also allows you to roll the 4 separate if checks into one.

Reply:
This has been fixed now